### PR TITLE
New headcrabs!

### PR DESCRIPTION
**Pull request by Terilia**
## What Does This PR Do
Adds new headcrabs to the game. We have the speedy headcrab, that can leap further and is generally faster, but a little weaker, and the poison headcrab, which will inject LSD into the victim with each bite. (5 units)

### **The sprites have been made by Ionward big thanks!!!**

Statistics for the number nerds:
Fast Headcrab
Health: 30 (default: 40)
Jumpcooldown: 30 (default: 40)
Jumpdistance: 8 (default: 4)
Jumpspeed: 2 (default: 1)

Poison Headcrab
Health: 60 (default: 40)
Jumpcooldown: 50 (default: 40)
Jumpdistance: 3 (default: 4)
Additional: adds 5 u LSD on biting their victim

It also adjusts the "headcrabs" event and will now automatically spawn either a normal headcrab group, a mix of fast and normal, a group of only fast, a group of poison and normal or a poison only group. The Spawn groups are:

Normal: Maximum 6
FastMix: Maximum 8
Fast: Maximum 6
PoisonMix: Maximum 4
Poison: Maximum 3

This does not mean that as many will spawn, it will still be between 2 and the maximum possible number. 

## Why It's Good For The Game
It gives admins a new random event to play with, and after some additional testing, a new random event that makes walking around in maintenance just a little more interesting than it currently is. 

## Images of changes
headcrab-variants_

## Changelog
:cl:
add: added new headcrabs (poison and fast)
soundadd: added new headcrab sounds for the poison headcrab and the fast zombie
/:cl:

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
we.. half-life now?

**Comment by Terilia** [comment]
I mean... yes

**Comment by JohnBigless** [comment]
> we.. half-life now?

Do you know who ate all the donuts?

**Comment by Leanfrog** [comment]
Why do we all have to wear these ridiculous ties

File: code/modules/mob/living/simple_animal/hostile/headcrab.dm [hunk 1]
```diff
@@ -104,3 +104,69 @@
...
+	icon_dead = "fast_headcrab_dead"
+	health = 30
+	maxHealth = 30
+	ranged_cooldown_time = 30
+	jumpdistance = 8
+	jumpspeed = 2
+	speak_emote = list("screech")
+
+/mob/living/simple_animal/hostile/headcrab/fast/update_icons()
+	..()
```
Comment (variableundefined) [inline code/modules/mob/living/simple_animal/hostile/headcrab.dm]: . = ..() please so it return its parent call

File: code/modules/mob/living/simple_animal/hostile/headcrab.dm [hunk 2]
```diff
@@ -104,3 +104,69 @@
...
+		overlays = human_overlays
+		var/image/I = image('icons/mob/headcrab.dmi', icon_state = "fast_headcrabpod")
+		if(host_species == "Vox")
+			I = image('icons/mob/headcrab.dmi', icon_state = "fast_headcrabpod_vox")
+		else if(host_species == "Gray")
+			I = image('icons/mob/headcrab.dmi', icon_state = "fast_headcrabpod_gray")
+		overlays += I
+
+/mob/living/simple_animal/hostile/headcrab/fast/Zombify(mob/living/carbon/human/H)
+	..()
```
Comment (variableundefined) [inline code/modules/mob/living/simple_animal/hostile/headcrab.dm]: Same as above

File: code/modules/mob/living/simple_animal/hostile/headcrab.dm [hunk 3]
```diff
@@ -104,3 +104,69 @@
...
+	health = 60
+	maxHealth = 60
+	ranged_cooldown_time = 50
+	jumpdistance = 3
+	jumpspeed = 1
+	attack_sound = 'sound/creatures/ph_scream1.ogg'
+	speak_emote = list("screech")
+
+/mob/living/simple_animal/hostile/headcrab/poison/update_icons()
+	..()
```
Comment (variableundefined) [inline code/modules/mob/living/simple_animal/hostile/headcrab.dm]: Same as above

File: code/modules/mob/living/simple_animal/hostile/headcrab.dm (hunk 1 above)
Comment (Terilia) [reply code/modules/mob/living/simple_animal/hostile/headcrab.dm]: should be implemented now

File: code/modules/mob/living/simple_animal/hostile/headcrab.dm (hunk 2 above)
Comment (Terilia) [reply code/modules/mob/living/simple_animal/hostile/headcrab.dm]: should be implemented now

File: code/modules/mob/living/simple_animal/hostile/headcrab.dm (hunk 3 above)
Comment (Terilia) [reply code/modules/mob/living/simple_animal/hostile/headcrab.dm]: should be implemented now

**Pull request merged**